apps/ima/sections/summary-data-sections.js

```diff
@@ -17,5 +17,20 @@ module.exports = {
         omitChangeLink: true
       }
     ]
+  },
+  'claim-details': {
+    steps: [
+      {
+        step: '/removal-condition',
+        field: 'how-removal-condition-1-applies',
+        parse: (list, req) => {
+          if (!req.sessionModel.get('steps').includes('/removal-condition')) {
+            return null;
+          }
+          return req.sessionModel.get('how-removal-condition-1-applies') !== '' ?
+            'Selected' + '\nDetails added' : 'Not selected';
```

Comment:
why do you need a new line?

Reply:
As part of the Figma design:

<img width="407" alt="image" src="https://github.com/UKHomeOffice/ima/assets/142597294/a7b528d9-f3dd-46e4-94cd-1086f8b74027">

Reply:
Fair enough